### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by wojtek-oledzki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,38 +7,54 @@ Develop: [![Build Status](https://travis-ci.org/sansible/newrelic_agent_java.svg
...
 
-To install run `ansible-galaxy install sansible.newrelic_agent_java` or add this to your
-`roles.yml`.
+To install run `ansible-galaxy install sansible.newrelic_agent_java` or add this to
+your `roles.yml`.
 
 ```YAML
 - name: sansible.newrelic_agent_java
-  version: v1.0
+  version: v1.0.0-latest
```
Comment (lobsterdore) [inline README.md]: What does the ```-latest``` but do?

File: tasks/configure.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- name: Configure New Relic Java agent
+  become: yes
+  template:
+    dest: "{{ sansible_newrelic_agent_java_install_dir }}/newrelic.yml"
+    group: "{{ sansible_newrelic_agent_java_group }}"
+    mode: "{{ sansible_newrelic_agent_java_configuration_file_mode }}"
+    owner: "{{ sansible_newrelic_agent_java_user }}"
+    src: newrelic.yml.j2
+  vars:
+    common: "{{ sansible_newrelic_agent_java_default_settings
```
Comment (lobsterdore) [inline tasks/configure.yml]: Might be good to mention these default settings in the README and how ```sansible_newrelic_agent_java_settings``` overrides them.

File: README.md (hunk 1 above)
Comment (wojtek-oledzki) [reply README.md]: an approach to make it semver naming compliant

File: tasks/configure.yml (hunk 2 above)
Comment (wojtek-oledzki) [reply tasks/configure.yml]: will do!

**Comment by wojtek-oledzki** [comment]
@lobsterdore have a look at README - added small section explaining `sansible_newrelic_agent_java_default_settings` vs `sansible_newrelic_agent_java_settings`

**Pull request merged**